middleware/allRoutes/handler.go

```diff
@@ -40,14 +40,21 @@ func Handler(routesHandler map[string]http.Handler, zebedeeClient *client.Client
 				return
 			}
 
-			// Construct contentPath with any colletion if present in cookie
+			if strings.HasSuffix(req.URL.String(), "/latest") {
```

Comment:
```suggestion
			if strings.HasSuffix(path, "/latest") {
```
would be more efficient and also more correct (ignore query string)

Reply:
I wonder if we could change line 36 (the previous `if` block) similarly - i.e. to use `path`

Reply:
I think there are cases where the file extension is in the query string. e.g. http://localhost:8888/file?uri=/peoplepopulationandcommunity/personalandhouseholdfinances/incomeandwealth/articles/earlyindicatorestimatesfromthewealthandassetssurvey/attitudestowardssavingforretirementcreditcommitmentsanddebtburdenjuly2014tojune2016/e04e4191.xls

I will be sure to take this point into account for further changes though 👍